modules/Hoot/ui/managePanel/jobs.js

```diff
@@ -691,7 +691,19 @@ export default class Jobs extends Tab {
                         let currentLayer = Hoot.layers.findBy( 'id', d.mapId );
 
                         if (currentLayer && !currentLayer.grailReference) {
-                            //Get info for the derive
+                            actions.push({
+                                title: 'derive changeset [adds only]',
+                                icon: 'add_to_photos',
+                                action: async () => {
+                                    const params  = {};
+                                    params.input1 = d.mapId;
+                                    params.parentId = d.jobId;
+
+                                    Hoot.api.deriveChangeset( params )
+                                        .then( resp => Hoot.message.alert( resp ) );
+                                }
+                            });
```

Comment:
Adds a new action to do single input derive changeset [adds only]
![image](https://user-images.githubusercontent.com/2600932/71174695-cf9beb80-2233-11ea-828c-fda8f823a362.png)
_*third from left_

It should be available to any import or conflate output that derive changeset replacement is, but doesn't need a reference layer to go with it.